crystal-map-connector/src/test/kotlin/ch/ergon/dope/extensions/type/relational/LikeTest.kt

```diff
@@ -16,20 +18,86 @@ class LikeTest : ManagerDependentTest {
     override lateinit var manager: DopeQueryManager
 
     @Test
-    fun `should support extensions for is like with CMJsonField`() {
-        val someCMStringField = someCMStringField()
-        val right = "%".toDopeType()
-        val expected = LikeExpression(someCMStringField.toDopeType(), right)
+    fun `should support extensions for is like with type cmString`() {
+        val left = someStringField()
+        val right = someCMStringField()
+        val expected = LikeExpression(left, right.toDopeType())
+
+        val actual = left.isLike(right)
+
+        assertEquals(expected.toDopeQuery(manager), actual.toDopeQuery(manager))
+    }
+
+    @Test
+    fun `should support extensions for is like with string cmString`() {
+        val left = someString()
+        val right = someCMStringField()
+        val expected = LikeExpression(left.toDopeType(), right.toDopeType())
+
+        val actual = left.isLike(right)
+
+        assertEquals(expected.toDopeQuery(manager), actual.toDopeQuery(manager))
+    }
+
+    @Test
+    fun `should support extensions for is like with cmString`() {
+        val left = someCMStringField()
+        val right = someString()
+        val expected = LikeExpression(left.toDopeType(), right.toDopeType())
+
+        val actual = left.isLike(right)
+
+        assertEquals(expected.toDopeQuery(manager), actual.toDopeQuery(manager))
+    }
+
+    @Test
+    fun `should support extensions for is like with cm`() {
```

Comment:
would change it to "...like with cmString cmString"

Reply:
done